.ci.yaml

```diff
@@ -2246,7 +2246,7 @@ targets:
 
   - name: Linux web_tool_tests
     recipe: flutter/flutter_drone
-    timeout: 60
+    timeout: 90 # https://github.com/flutter/flutter/issues/169634
```

Comment:
Keep this and the `test_timeout_secs` property below in sync.

Reply:
Done.